qiskit_ibm_provider/hub_group_project.py

```diff
@@ -13,19 +13,17 @@
 """A hub, group and project in an IBM Quantum account."""
```

Comment:
Overwritten with most recent version of https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/hub_group_project.py